package.json

```diff
@@ -32,9 +33,15 @@
   ],
   "license": "MIT",
   "peerDependencies": {
-    "react-native": ">=0.16.0"
+    "react-native": ">=0.16.0",
```

Comment:
Not sure if this is valid. RN 25+ introduced a number of breaking changes. Maybe this is better as >=0.25.0

https://github.com/appintheair/react-native-looped-carousel/commit/a74c53519b9793e890c404c46a220bbe59d46b5a